slack-pi-command/function.js

```diff
@@ -0,0 +1,25 @@
+const { parse } = require('query-string');
+const PI = require('pi');
+
+exports.handler = async (body, context) => {
+  // slack unfortunately sends command payloads as x-url-form-encoded
```

Comment:
1. Because it's not the native type you would expect for these sort of things (especially considering that all Slack "bots" receive their payloads as JSON).
2. I'm not sure it's important, the API is well defined.

Reply:
👍